### PR TITLE
Nikhil's Yearbook

### PR DESCRIPTION
**Pull request by nikhiltripathi1**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by darkRaspberry** [comment]
Your check list is incorrect.

Simply put a x in between [ ]
don't put any square between [x] these
It should change in tick

File: _data/nikhiltripathi1/nikhiltripathi1.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Nikhil Tripathi # No longer than 28 characters
+institution: Dr. Akhilesh Das gupta Institute of Technology and Management # no longer than 58 characters
+profile_pic: nikhil.png # Name and extension of your profile picture(ex. mona.png)
+quote: exploring or escaping, what should we choose? # no longer than 100 characters
+github_user: nikhiltripathi1
+---
```
Comment (darkRaspberry) [inline _data/nikhiltripathi1/nikhiltripathi1.md:7]:
```suggestion
name: Nikhil Tripathi
institution: Dr. Akhilesh Das gupta Institute of Technology and Management
profile_pic: nikhil.png
quote: exploring or escaping, what should we choose?
github_user: nikhiltripathi1
---
```

Comment (ShaunLWM) [reply _data/nikhiltripathi1/nikhiltripathi1.md]: I think having those comments in the markdown file is fine though. Most of them already got approved and pushed to the repository.

**Comment by nikhiltripathi1** [comment]
i have updated my .md file and how do i fix my checklist?

**Comment by nikhiltripathi1** [comment]
@darkRaspberry @SulthanNK @jeffrafter @ShaunLWM please verify it, i have updated my markdown file and aslo verified the checklist

**Pull request closed without merging**